docs/modules/ROOT/pages/guides/v3-migration/index.adoc

```diff
@@ -0,0 +1,159 @@
+
+Note that any other notation, such as `[Actor]` or `[Actor!]` will **not** be valid. "One to one" relationships
+such as `director` remain unchanged.
+
+=== Relationship cardinality
+Runtime checks for relationships number of connections have been rolled in with this release, this means that some
+databases with inconsistent relationships between the schema definition and the actual data may now fail in some queries.
+For those cases, please ensure that the database is following your schema definition or update the schema to reflect the
+actual existing relationships.
+
+== Count query no longer supported
+Queries using `count` at the root level are no longer supported. For example:
+[source, graphql, indent=0]
+----
+Query {
+    usersCount
+}
+----
+
+The same operation, can now be achieved with a xref::queries#_counting_using_aggregation[count aggregation] query:
+
+[source, graphql, indent=0]
+----
+Query {
+    usersAggregate {
+        count
+    }
+}
+----
+
+== Relationship filters
+`where` filters for relationship queries now explicitly state `ALL`, `NONE`, `SINGLE`, and `SOME` as part of filter name.
+
+Queries using old relationship filters, will now need to use `{relationship}_NONE`. For example:
+
+[source, graphql, indent=0]
+----
+Query {
+  movies(where: {
+    actors: {
+      name: "John"
+    }
+  }) {
+    title
+  }
+}
+----
+
+Should be:
+
+[source, graphql, indent=0]
+----
+Query {
+  movies(where: {
+    actors_SOME: {
+      name: "John"
+    }
+  }) {
+    title
+  }
+}
+----
+
+And, instead of `_NOT`, `_NONE` should be used.
+
+NOTE: Old queries will still work in this release, but are marked as `@deprecated` and will not be available in the future.
+
+== `@ignore` directive renamed to `@computed`
+To better reflect its intended usage, the `@ignore` directive is now named `@computed`, behaviour is unchanged, so you just need to
+rename this directive in your schema.
+
+== Auth plugin system
+https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-key-sets[JSON Web Key Sets] support through `jwksEndpoint` option is no longer
+provided directly by `@neo4j/graphql`.
+
+This feature is now available as a plugin `@neo4j/graphql-plugin-auth` that can be installed if needed:
+
+[source, JavaScript, indent=0]
+----
+const { JWTPlugin } = require("@neo4j/graphql-plugin-auth");
+
+const neoSchema = new Neo4jGraphQL({
+     typeDefs,
+     plugins: {
+         jwt: new JWTPlugin({
+             secret: "secret",
+         }),
+     },
+ });
+----
+
+== Types plurals changes
+To improve consistency, some automatically generated plurals (e.g. `createActors`) have changed. This may cause issues if
+your types use conventions such as `snake_case`.
+
+Because of this, you may find generated queries and mutations may have different names. If you encounter this problem,
+please update your clients to use the new query names or use the `plural` option in the xref::type-definitions/database-mapping.adoc#_plural[`@node directive`]
```

Comment:
More of an accessibility note. The link to `@node directive` is very helpful but it wasn't very obvious to me that this is a link.

Reply:
What alternative could we use? just "@node directive" without code blocks?

Reply:
The alternative without code blocks seems good. But on the other hand this may also just be some useless nitpicking from my side 😅 . We can leave it as is.

Reply:
If you think it helps accessibility I'm happy to change it
```suggestion
please update your clients to use the new query names or use the `plural` option in the xref::type-definitions/database-mapping.adoc#_plural[@node directive]
```